.circleci/config.yml

```diff
@@ -136,13 +148,21 @@ workflows:
   version: 2
   build-and-publish:
     jobs:
-      - build
+      - build:
+          context:
+            - dockerhub-read
       - integration-test
```

Comment:
The integration tests run on a machine executor that's using docker (not remote) - need to log in there and provide context too.

Reply:
fixed.